pip/req/req_install.py

```diff
@@ -715,32 +714,18 @@ def uninstall(self, auto_confirm=False):
                     paths_to_remove.add(os.path.join(bin_dir, script) + '.bat')
 
         # find console_scripts
-        if dist.has_metadata('entry_points.txt'):
-            if six.PY2:
-                options = {}
+        console_scripts = dist.get_entry_map(group='console_scripts')
```

Comment:
Good point, I'm wondering if the gui_scripts ar correctly uninstalled...
But I guess they should be in the `installed-files.txt` file.